Tuist/ProjectDescriptionHelpers/Targets/Target+ApolloPaginationTests.swift

```diff
@@ -19,7 +19,8 @@ extension Target {
                 .package(product: "Apollo"),
                 .package(product: "ApolloAPI"),
                 .package(product: "ApolloTestSupport"),
-                .package(product: "ApolloPagination")
+                .package(product: "ApolloPagination"),
+                .package(product: "Nimble")
```

Comment:
👀 
Nimble isn't in use here yet, as far as I can tell. Excited to see where and how it gets used!

Reply:
Yeah, I planned on using it in some tests for the original approach, but then when we went with this approach, I deleted those tests. But regardless, it's we've always used for test matchers and will use in the future, so it's fine to add it here for now.